test/Microsoft.AspNet.Mvc.FunctionalTests/LocalizationTest.cs

```diff
@@ -71,5 +74,100 @@ public static IEnumerable<object[]> LocalizationData
             // Assert
             Assert.Equal(expected, body.Trim());
         }
+
+        public static IEnumerable<object[]> LocalizationResourceData
+        {
+            get
+            {
+                var expected1 =
+ @"My ASP.NET Application
+
+Hello there!!
+Learn More";
+
+                yield return new[] {"en-GB", expected1 };
+
+                var expected2 =
+ @"Mon application ASP.NET
+
+Bonjour!
+apprendre Encore Plus";
+
+//Coreclr just reads enu resources. Hence the conditional if
```

Comment:
This is a hack since dnx does not support reading non-enu resource files. Once we have the support for it, we can remove this code and mocked resource manager code too.